maggy/ablation/ablator/loco.py

```diff
@@ -203,7 +218,11 @@ def create_trial_dict(self, ablated_feature=None, layer_identifier=None):
         :rtype: dict
         """
 
+        #TODO modify trial dict structure: rename 'ablated_layer' to 'model configuration'
```

Comment:
Are you going to address this in this PR?

Reply:
No, this will go in a separate PR.